src/reducers/books.js

```diff
@@ -0,0 +1,50 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actionTypes';
+
+const listingBooks = [
+  {
+    id: 1,
+    title: 'A Promised Land',
+    category: 'Biography',
+  },
+  {
+    id: 2,
+    title: 'Ambitious Girl',
+    category: 'Kids',
+  },
+  {
+    id: 3,
+    title: 'Love you Forever',
+    category: 'Kids',
+  },
+  {
+    id: 4,
+    title: 'Good Night Moon',
+    category: 'Sci-Fi',
+  },
+  {
+    id: 5,
+    title: '1922',
+    category: 'Horror',
+  },
+];
+
+function removeBookFromState(id) {
+  return function newState(book) {
```

Comment:
## Convert to ES6 syntax
- [ ] We need to ensure we are using ES6 syntax all throughout the codebase. Kindly convert this `newState` reducer function declaration to an arrow based function.
```suggestion
  return const newState = (book) => {
```